staff/bernat-casasus/restaurants-app.1/.versions/restaurants-app-1.0.0/scripts/web-components-1.0.0.js

```diff
@@ -0,0 +1,153 @@
+'use strict';
+
+/**
+ * Web Components lib
+ * 
+ * @author manuelbarzi
```

Comment:
no ets el autor del teu web-components?

Reply:
Please If posible use english to send comments!